### ISSUE TITLE
discord.js example segfaults on windows

### ISSUE THREAD

**Issue by paperclover** [oven-sh/bun#9062]
```ts
const { Client, Events, GatewayIntentBits } = require("discord.js");

const client = new Client({ intents: [GatewayIntentBits.Guilds] });

client.once(Events.ClientReady, (readyClient) => {
  console.log(`Ready! Logged in as ${readyClient.user.tag}`);
});

client.login(process.env.token);
```

```
Segmentation fault at address 0x28054f74ff0
C:\bun\packages\bun-usockets\src\crypto\openssl.c:499:0: 0x7ff6855bf581 in ssl_on_data (openssl.c.obj)
    s = (struct us_internal_ssl_socket_t *)context->sc.on_writable(

C:\bun\packages\bun-usockets\src\loop.c:344:0: 0x7ff6855c75b6 in us_internal_dispatch_ready_poll (loop.c.obj)
                        s = s->context->on_data(s, loop->data.recv_buf + LIBUS_RECV_BUFFER_PADDING, length);

C:\bun\packages\bun-usockets\src\eventing\libuv.c:27:0: 0x7ff6855c5eb0 in poll_cb (libuv.c.obj)
  us_internal_dispatch_ready_poll((struct us_poll_t *)p->data, status < 0,

???:?:?: 0x7ff685879989 in ??? (bun-debug.exe)
???:?:?: 0x7ff68586b045 in ??? (bun-debug.exe)
???:?:?: 0x7ff68586a98f in ??? (bun-debug.exe)
C:\bun\packages\bun-usockets\src\eventing\libuv.c:201:0: 0x7ff6855c63c5 in us_loop_run (libuv.c.obj)
  uv_run(loop->uv_loop, UV_RUN_ONCE);

C:\bun\src\deps\uws.zig:2541:20: 0x7ff682b53d52 in run (bun-zig.o)
        us_loop_run(this);
                   ^
C:\bun\src\bun.js\event_loop.zig:1330:22: 0x7ff682904562 in autoTickActive (bun-zig.o)
            loop.tick();
                     ^
C:\bun\src\bun_js.zig:404:50: 0x7ff682e98916 in start (bun-zig.o)
                    vm.eventLoop().autoTickActive();
                                                 ^
C:\bun\src\bun.js\javascript.zig:105:21: 0x7ff6829edcd5 in callback (bun-zig.o)
            Function(context);
                    ^
```

**Comment by dylan-conway** [comment]
most likely fixed by #9193

**Comment by cirospaciari** [comment]
fixed by #9191
